packages/components/tests/dummy/app/templates/components/text.hbs

```diff
@@ -0,0 +1,123 @@
+
+<section data-test-percy>
+
+  <Shw::Text::H2>Variants</Shw::Text::H2>
+  <Shw::Text::Body><em>Notice: we are showing only the combinations of
+      <code>font-size</code>
+      ("style") and
+      <code>font-weight</code>
+      that the design system
+      <strong>suggests</strong>
+      to use.</em></Shw::Text::Body>
+
+  <Shw::Text::H4 @tag="h3">Text::Display</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.DISPLAY_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.DISPLAY_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Display @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Display>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Text::H4 @tag="h3">Text::Body</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.BODY_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.BODY_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Body @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Body>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Text::H4 @tag="h3">Text::Code</Shw::Text::H4>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each this.model.CODE_AVAILABLE_SIZES as |size|}}
+      <SF.Item @label="size={{size}}">
+        {{#let (get this.model.CODE_AVAILABLE_WEIGHTS_PER_SIZE size) as |weights|}}
+          {{#each weights as |weight|}}
+            <Hds::Text::Code @size={{size}} @tag="p" @weight={{weight}}>The fox jumped over the lazy dog ({{weight}})</Hds::Text::Code>
+          {{/each}}
+        {{/let}}
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Alignment</Shw::Text::H2>
+  <Shw::Grid @columns={{3}} as |SG|>
+    {{#each this.model.AVAILABLE_ALIGNS as |alignment|}}
+      <SG.Item>
+        <Shw::Flex @direction="column" as |SF|>
+          <SF.Item @label="parent with text-align={{alignment}}">
+            <Shw::Outliner {{style text-align=alignment}}>
+              <Hds::Text::Body @size="200" @tag="p">The fox jumped over the lazy dog</Hds::Text::Body>
+            </Shw::Outliner>
+          </SF.Item>
+          <SF.Item @label="with @align={{alignment}}">
+            <Shw::Outliner>
+              <Hds::Text::Body @size="200" @tag="p" @align={{alignment}}>The fox jumped over the lazy dog</Hds::Text::Body>
+            </Shw::Outliner>
+          </SF.Item>
+        </Shw::Flex>
+      </SG.Item>
+    {{/each}}
+  </Shw::Grid>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H2>Color</Shw::Text::H2>
+
+  <Shw::Text::H4 tag="h3">Color inheritance</Shw::Text::H4>
```

Comment:
I don't think this is rendering the way you intend it to render: 
![CleanShot 2023-07-27 at 13 16 59](https://github.com/hashicorp/design-system/assets/4587451/c0416ddc-34ad-485e-a3e0-aa418f7ba2bc)
(and it's causing a failure since heading levels should only increase by one)